equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm5Controller.java

```diff
@@ -2,13 +2,15 @@
 
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import uk.gov.justice.laa.crime.equinity.historicaldata.generated.api.Crm5InterfaceApi;
 import uk.gov.justice.laa.crime.equinity.historicaldata.generated.dto.CRM5DetailsDTO;
 import uk.gov.justice.laa.crime.equinity.historicaldata.service.CrmFileService;
 
 @RestController
 @RequiredArgsConstructor
+@RequestMapping("/api/internal/v1/equinity")
```

Comment:
This should be moved out and set as part of the Open API specification. That way it is part of the Interface. I persume it will be something like: `/api/internal/vi/equinity/crm5/{usn}`

Reply:
will be done